book/src/attributes/logos.md

```diff
@@ -34,7 +34,8 @@ The type `ErrorType` can be any type that implements `Clone`, `PartialEq`,
 `Default` and `From<E>` for each callback's error type.
 
 `ErrorType` must implement the `Default` trait because invalid tokens, i.e.,
-literals that do not match any variant, will produce `Err(ErrorType::default())`.
+literals that do not match any variant, will produce `Err(ErrorType::default())`,
+unless you provide a callback with the alternate syntax `#[logos(error(ErrorType, callback = ...))]`
```

Comment:
Sorry for the late review. Just one last comment: can you add a bit more details about the callback (signature) and maybe link to the example you wrote? Otherwise, users will not know it is documented :-)

Otherwise, it's all good to be merged!

Reply:
I actually just changed the custom_error example to use the new error syntax instead of adding a new one, and the example is already linked in the book